sys/include/net/ieee802154.h

```diff
@@ -43,28 +55,146 @@ extern "C" {
 #define IEEE802154_FCF_TYPE_ACK             (0x02)
 #define IEEE802154_FCF_TYPE_MACCMD          (0x03)
 
-#define IEEE802154_FCF_SECURITY_EN          (0x08)
-#define IEEE802154_FCF_FRAME_PEND           (0x10)
-#define IEEE802154_FCF_ACK_REQ              (0x20)
-#define IEEE802154_FCF_PAN_COMP             (0x40)
+#define IEEE802154_FCF_SECURITY_EN          (0x08)  /**< enable security */
+#define IEEE802154_FCF_FRAME_PEND           (0x10)  /**< follow-up frame is pending */
+#define IEEE802154_FCF_ACK_REQ              (0x20)  /**< acknowledgement requested from receiver */
+#define IEEE802154_FCF_PAN_COMP             (0x40)  /**< compress source PAN ID */
 
 #define IEEE802154_FCF_DST_ADDR_MASK        (0x0c)
-#define IEEE802154_FCF_DST_ADDR_VOID        (0x00)
-#define IEEE802154_FCF_DST_ADDR_SHORT       (0x08)
-#define IEEE802154_FCF_DST_ADDR_LONG        (0x0c)
+#define IEEE802154_FCF_DST_ADDR_VOID        (0x00)  /**< no destination address */
+#define IEEE802154_FCF_DST_ADDR_RESV        (0x04)
```

Comment:
Do we need a constant for a reserved field? If yes, why not document it with `/**< reserved */`?